.github/workflows/dluhc-build-and-publish.yml

```diff
@@ -13,7 +13,7 @@ on:
       ]
 
 env:
-  VERSION: "0.1.138" # Manually increment this version when pushing to main
+  VERSION: "0.1.139" # Manually increment this version when pushing to main
```

Comment:
@adamdavies1 This PR that got merged already has V139:
https://github.com/communitiesuk/digital-form-builder/pull/317/files

Reply:
will bump further

Reply:
@adamdavies1 PR Approved